x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/persistence/TransformInternalIndex.java

```diff
@@ -399,7 +405,7 @@ protected static void installLatestAuditIndexTemplateIfRequired(
 
         // Installing the template involves communication with the master node, so it's more expensive but much rarer
         try {
-            IndexTemplateMetaData indexTemplateMetaData = getAuditIndexTemplateMetaData();
+            IndexTemplateMetaData indexTemplateMetaData = getAuditIndexTemplateMetaData(clusterService.state().nodes().getMinNodeVersion());
```

Comment:
FWIW, for later readers: I think this is the important change of this PR as this is the place where a (potentially new) non-master calls the (potentially old) master.